product_code/lagom/hl_course_service/impl/src/main/resources/application.conf

```diff
@@ -0,0 +1,41 @@
+#
+#
+play.application.loader = de.upb.cs.uc4.hl.course.impl.HlCourseLoader
+
+db.default {
```

Comment:
The hl_course_service doesn't use Postgres, no driver definition needed

Reply:
Remove whole db.default?

Reply:
Yes

Reply:
removed by new commit